### PR TITLE
Updated changelog for 0.4.6

### PR DESCRIPTION
**Pull request by Vezzra**

### CODE REVIEW

**Comment by Vezzra** [comment]
I've made a PR out of the first draft of the changelog for 0.4.6 @geoffthemedio [posted on the forum](http://www.freeorion.org/forum/viewtopic.php?p=85385#p85385). As we traditionally commit changelog updates to the release branch and cherry-pick them into master after the release, I created this PR against the release branch.

I also created the changelog branch in the main repo, not my clone, so we can better collaborate on it.

I made some small changes to the text @geoffthemedio posted:
- I reordered the items in the "Key Changes" section and extended the item about the GUI improvements.
- Fixed a typo.
- Moved the entry "In DesignWnd add ability to drag parts away from design" from the "Balance" section to the "Graphics/GUI" section, assuming that it somehow erroneously ended up in "Balance" (because IMO this hasn't anything to do with balance, but everything with the GUI).
  Unless you had a specific reason to put it in "Balance" that I missed - @geoffthemedio?
- Rearranged order of a few items.

@geoffthemedio, are these adjustments ok with you? Feel free to revert any of them you don't think fitting.

I have some further comments I will post below, partially as comments to specific lines. Everyone else, please also review the changelog and post your corrections, either here as comments or provide PRs.

**Comment by geoffthemedio** [comment]
tweak as you want, I'll complain if I don't like it and notice

File: ChangeLog.md [hunk 1]
```diff
@@ -3,6 +3,308 @@
...
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - New/better graphics
```
Comment (geoffthemedio) [inline ChangeLog.md]: this does not seem like a "key change"

File: ChangeLog.md [hunk 2]
```diff
@@ -3,6 +3,308 @@
...
+ - Lot of other small improvements to handling/display of things
```
Comment (geoffthemedio) [inline ChangeLog.md]: Perhaps make these a separate top-level point, rephrased to something like 

```
Many small changes to GUI rendering, layout, customization options, and displayed information that collectively substantially improve usability, responsiveness, and FPS or system resource usage.
```

File: ChangeLog.md [hunk 3]
```diff
@@ -3,6 +3,310 @@
...
+- Added 3D starfield effect on map
+- Add an entry on the right-click menu to add to top of build queue
+- When left-clicking production queue items, open the system and
+select the planet instead of just jumping to the system on the map.
+- Made ctrl-left click remove an item from build queue.
+- Made ctrl-double click add entries to the top of the build queue
+- Pedia articles can now have images in them
+- New icons for auto and manual turn cycling button
+- Tweaked Sidepanel plant pole tilt angle
+- Tweaked Sitrep layout
```
Comment (LGM-Doyle) [inline ChangeLog.md:63]: plant -> planet

File: ChangeLog.md [hunk 4]
```diff
@@ -3,6 +3,310 @@
...
+#### Balance
+
+- Various building / tech cost adjustments
+- Split Ground Troops species trait into Defensive Ground Troops and
+Offensive Ground Troops.
+- Made Neutronium Forge only be buildable at locations with a Basic
+Shipyard (like all other shipyard upgrade buildings)
+- Tweaked monster detection and stealth stats
+- Artificial Moons can no longer be build when a Resonant Moon is present
+- Reworked various modifiers to planet population
```
Comment (LGM-Doyle) [inline ChangeLog.md:228]: build->built

File: ChangeLog.md [hunk 5]
```diff
@@ -3,6 +3,310 @@
...
+ - Empires' supply ranges push against eachother in neutral systems,
+with higher range supply sources blocking lower range (where range
+decays with distance from the source)
+ - Armed ships prevent enemy supply from propagating in/out of a system
+ - In the absence of armed ships, supply won't propagate into an enemy
+system where a supply source planet is located
+- Made producing troop pods require a production location with troops,
+and made producing troop pods consume troops from the production
+location
+ - Species with no offensive Ground Troops will no longer be able to
```
Comment (LGM-Doyle) [inline ChangeLog.md:242]: Wasn't the finite troops feature removed pending reworking?

**Comment by LGM-Doyle** [comment]
Look good to me.

An impressive amount of work.

Congrats.

File: ChangeLog.md [hunk 6]
```diff
@@ -3,6 +3,308 @@
...
+production between planets.
+ - Empires' supply ranges push against eachother in neutral systems,
+with higher range supply sources blocking lower range (where range
+decays with distance from the source)
+ - Armed ships prevent enemy supply from propagating in/out of a system
+ - In the absence of armed ships, supply won't propagate into an enemy
+system where a supply source planet is located
+- Made producing troop pods require a production location with troops,
+and made producing troop pods consume troops from the production
+location
```
Comment (Vezzra) [inline ChangeLog.md]: Anyone please correct my if I'm wrong, but is that bit (about producing troop pods consuming troops from the production location) actually the case? In my recent test games I didn't notice that happening, and the relevant effects in the troop pod ship parts are commented out. Unless these effects have been added elsewhere?

IIRC that happened because with that change alone, producing enough troop ships became almost impossible, and we started a discussion about how to address the issue. As we didn't reach a general consensus yet, this feature has been disabled, at least that's how I remember it.

So that part needs to be taken out of the changelog, right?

File: ChangeLog.md [hunk 7]
```diff
@@ -3,6 +3,308 @@
...
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - New/better graphics
+ - Lot of other small improvements to handling/display of things
+- Turn processing time improvements
+- AI improvements
+
```
Comment (geoffthemedio) [inline ChangeLog.md:24]: Maybe add

```
New specials, hulls, ship parts, giving new strategic options or fun gameplay options
```

File: ChangeLog.md (hunk 5 above)
Comment (geoffthemedio) [reply ChangeLog.md:242]: When prepping I wasn't sure about that, but couldn't easily check

Comment (MatGB) [reply ChangeLog.md:242]: The troop pods costing troops aspect was commented out as it requires an AI rewrite and some balancing, but the requirements for the planet to have a troop capacity wasn't commented out.

File: ChangeLog.md [hunk 8]
```diff
@@ -3,6 +3,308 @@
...
+take effect requires being in supply range)
+- Added Energy Frigate hull, adjusted some other hulls' balance accordingly
+- Updated premade / default ship designs, made naming consistent
+- New bombardment weapons (feature in-progress)
+- Added EMP and electric overcharge ship parts
+- Added Solar Concentrators which increase laser weapon strength
+depending on local star type
+- Split up stealth part and planetary stealth techs
+- Added Flak Cannon ship part that attacks multiple times per combat
+turn but does minimal damage per attack
```
Comment (Vezzra) [inline ChangeLog.md]: Again, please, anyone, correct me if I'm wrong, but I don't think that ship part is actually available in the game yet. The item is already present in the content scripts, but AFAIK doesn't get unlocked. It's part of the entire fighters/carrier stuff that has been postponed until after the release.

So I guess that should be removed from the changelog too?

**Comment by Vezzra** [comment]
I might have missed it when reading that massive list of changes, but I didn't find an entry that mentioned the following: all systems that are not deep space, black holes or neutron stars now should have at least one planet/asteroid belt (so there are no more systems with stars, but no planets).

If it's really missing, should be added.

File: ChangeLog.md (hunk 1 above)
Comment (Vezzra) [reply ChangeLog.md]: Agreed.

**Comment by geoffthemedio** [comment]
Don't recall adding anything like that

File: ChangeLog.md (hunk 2 above)
Comment (Vezzra) [reply ChangeLog.md]: Basically agree, but as the FPS improvements are already mentioned above, I'd drop that specific bit (and just leave "...and system resource usage").

Another thing I tried to cover by using the term "handling" are the many small additions to GUI functionality (like dismissing sensor ghosts etc.). So maybe phrase it like this:

```
Many small additions/changes to GUI rendering, layout, functionality, customization options, and displayed information that collectively substantially improve usability, responsiveness, and system resource usage.
```

...?

File: ChangeLog.md (hunk 3 above)
Comment (Vezzra) [reply ChangeLog.md:63]: Good catch, thx!

File: ChangeLog.md (hunk 5 above)
Comment (Vezzra) [reply ChangeLog.md:242]: Well, that answers my question below 😃

File: ChangeLog.md [hunk 9]
```diff
@@ -3,6 +3,308 @@
...
+- Translation updates: French, Russian
+- Removed / updated various missing, unused, or incorrect stringtable entries
+- Adjusted AI priorities for various buildings
+- Adjusted AI colonization planning
+- Made AI more cautious of hidden threats after losing fleets in a
+system it can't detect into
+- Made AI adjust priorities based on costs of ship parts and hulls
+- Adjusted AI priorities for research time
+- Adjusted AI industry modifiers
+- Adjusted AI weapon upgrade calculations
```
Comment (Morlic-fo) [inline ChangeLog.md]: Except for the losing-fleet part, imo the rest of the AI changes are small and unspecific enough to be summarized as "Various adjustments to AI priorities".

File: ChangeLog.md [hunk 10]
```diff
@@ -3,6 +3,308 @@
...
+- Added sitreps to Planetary Starlane Drive effects
+- Various new hull art and slot positions, tech icons, building icons
+- Added the Accretion Disc special
+- Fixed / updated bioweapon sitreps
+- Added a random turn event that creates Space Krill in an empty system
+- Added new galaxy shape "Disc"
+- Added the feature to resurrect extinct species found in ancient
+ruins with the Xenoresurrection Lab
+- Added the new special Temporal Anomaly
+- Reworked weapon part refinements, so that instead of multiple parts,
```
Comment (Morlic-fo) [inline ChangeLog.md:177]: IMO, this is a quite significant gameplay change that does not deserve to be buried so deeply.

File: ChangeLog.md (hunk 4 above)
Comment (Vezzra) [reply ChangeLog.md:228]: Again, good catch, thanks!

File: ChangeLog.md [hunk 11]
```diff
@@ -3,6 +3,308 @@
...
+depending on local star type
+- Split up stealth part and planetary stealth techs
+- Added Flak Cannon ship part that attacks multiple times per combat
+turn but does minimal damage per attack
+- Added more AI greeting messages, made random
+- Added Scrying Sphere buildings, which can be produced and appear
+around the map randomly during galaxy generation, and share vision
+between planets on which they are located
+- The Honeycomb special will now turn nearby planets into asteroids
+instead of destroying them
```
Comment (Morlic-fo) [inline ChangeLog.md]: Maybe this could be rewritten from the actual player perspective, i.e. instead of no planets in 5(`?) jump radius, there may only be asteroids?

File: ChangeLog.md [hunk 12]
```diff
@@ -3,6 +3,308 @@
...
+
+#### Balance
+
+- Various building / tech cost adjustments
+- Split Ground Troops species trait into Defensive Ground Troops and
+Offensive Ground Troops.
+- Made Neutronium Forge only be buildable at locations with a Basic
+Shipyard (like all other shipyard upgrade buildings)
+- Tweaked monster detection and stealth stats
+- Artificial Moons can no longer be build when a Resonant Moon is present
```
Comment (Morlic-fo) [inline ChangeLog.md]: `Resonant Moon special` ?

File: ChangeLog.md [hunk 13]
```diff
@@ -3,6 +3,308 @@
...
+- Re-Added the Tidally Locked Rotation special with a population
+penalty and changed the penalty of the Eccentric Orbit special to a
+Supply minus
+- Space Monster weapon balance tweaks
+- Planetary Starlane Drive modified so it can't be built on gas giants
+- Xenoresurrection Lab building modified so that only one can be built
+on a planet
+- Death Spores and Bio-Terminators now only affect organic species
+- Monsters now only bombard one planet at a time
+- Adjusted Experimentor monster weapons
```
Comment (Morlic-fo) [inline ChangeLog.md]: Can / Should be grouped with the ```Space Monster weapon balance tweaks" 6 lines above?

File: ChangeLog.md (hunk 10 above)
Comment (Vezzra) [reply ChangeLog.md:177]: Seconded, I also wanted to suggest that for inclusion into the "Key Changes" section.

File: ChangeLog.md [hunk 14]
```diff
@@ -3,6 +3,308 @@
...
+location
+ - Species with no offensive Ground Troops will no longer be able to
+build ships with Ground Troop pods
+- Balanced research and production bonuses
+- Reprioritised mine effects to always occur before repair effects
+- Balance pass on Robotic line and Damage Control techs
+- Balanced BioAdaptive hull to work same as Nanorobotic
+- Tweaked numbers for Logistics Facilitator
+- Prevent Krakens Nests from spawning on Asteroid belts created by the
+Honeycomb special (and similar situations)
```
Comment (Morlic-fo) [inline ChangeLog.md]: Considering that the Honeycomb special was reworked in this version to actually create those Asteroids, does this bugfix really belong into the changelog?

File: ChangeLog.md (hunk 11 above)
Comment (geoffthemedio) [reply ChangeLog.md]: I wasn't familiar enough with the special to know that that wasn't from a player perspective. If it's just something that happens at game start and not during a game when a player can see it, that would be a reasonable change.

File: ChangeLog.md (hunk 14 above)
Comment (geoffthemedio) [reply ChangeLog.md]: probably not

File: ChangeLog.md (hunk 6 above)
Comment (Ouaz) [reply ChangeLog.md]: Yes, `and made producing troop pods consume troops from the production location` must be removed.
I've just checked in-game, no troops are consumed when a ship with a Troop Pod part is built (only the minimum Troop meter value is required - 2 or 4 depending the Troop Pod class - basic or advanced)

File: ChangeLog.md (hunk 7 above)
Comment (Vezzra) [reply ChangeLog.md:24]: Good idea.

File: ChangeLog.md (hunk 8 above)
Comment (Ouaz) [reply ChangeLog.md]: Yes again. ^^

Actually, the Flak Cannon ship part is displayed in the Design Window (with Unavailable filter on), but there is no way to unlock it (no tech)

Comment (Morlic-fo) [reply ChangeLog.md]: Speaking of which, should those fighter-related items (flak cannon, hangars, bay) be disabled for release?

**Comment by Vezzra** [comment]
Ok, before I call it a day, a list of suggestion for the "Key Changes" section:
- As @Morlic-fo suggested above, the reworked weapon upgrade mechanic is a prime candidate (IMO).
- The reworking/rebalancing of Orbital Drydock repair and Damage Control Techs was one of the bigger changes in my eyes, but not sure if big enough for a "key change"? Opinions?
- Ditto for the new bombardment weapons (at least from the players perspective this could be seen as one of the bigger additions?) and the reorganization/rearrangement of the FOCS scripts (for content scripters splitting up the huge content files and rearranging everything into a directory tree probably is a major change?). Again, opinions?

I'll continue committing all the suggestions/corrections you made tomorrow (although, committers, feel free to commit stuff that has been agreed upon or is obvious 😉).

**Comment by geoffthemedio** [comment]
Regrouping the organization of script files is big change for scripters, but not players.
Damage control techs that only affect a subset of ship types (robotic?) is probably a bit narrow for a key change (vs. Supply mechanics that are relevant to almost any player) but OK if others think it's a big deal.

**Comment by dbenage-cx** [comment]
Not seeing a mention of the change to user directory location (linux) 4206b7a
That and content definition file restructure bear a mention in technical.

Agree the other content changes mentioned are not key (drydock, damage control, bombardment, restructure of files).

File: ChangeLog.md [hunk 15]
```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
```
Comment (MatGB) [inline ChangeLog.md]: "and introduced system-control circle colours, only one empire can control/supply a system"

It's a very important and significant change, it needs explaining.

File: ChangeLog.md [hunk 16]
```diff
@@ -3,6 +3,309 @@
...
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - Lot of other small improvements to handling/display of things
+- Turn processing time improvements
+- AI improvements
+- New specials, hulls, ship parts, giving new strategic or
+fun gameplay options
+
```
Comment (MatGB) [inline ChangeLog.md:24]: Weapon Refinements, use Geoff's text, it's a very significant change and a substantial improvement.

File: ChangeLog.md [hunk 17]
```diff
@@ -3,6 +3,309 @@
...
+
+### Key Changes
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - Lot of other small improvements to handling/display of things
+- Turn processing time improvements
```
Comment (MatGB) [inline ChangeLog.md:21]: "and in game lag reduction"

It really is a substantial improvement and it's been a fairly common complaint from players for ages.

File: ChangeLog.md [hunk 18]
```diff
@@ -3,6 +3,309 @@
...
+progress meters
+- Added specialized tooltip for resource icons
+- Made encyclopedia detail panel of production and research screens closable
+- Switched a few objects list columns from showing object ids to object names
+- Added nearest system column and number of specials column to objects list
+- Production and research queue items show max spendable PP or RP per turn
+- Added ship rally point tracking / setting to production queue. New
+fleets are grouped by rally point on the turn they are produced. Ship
+production items can be right clicked to set their rally point to the
+selected system.
```
Comment (MatGB) [inline ChangeLog.md:83]: This should be in Key Changes, or at least a summary of it, it's very useful and has been requested several times.

File: ChangeLog.md (hunk 8 above)
Comment (MatGB) [reply ChangeLog.md]: Don't disable them, for a start a few monsters use Flak Cannons, I think they should be mentioned but it made clear they're not player-usable yet. IIRC the Drone Factory, Warden and Sentinel use them.

I have no opinion on whether hangers et al should be disabled, but I don't see the need to do so.

File: ChangeLog.md [hunk 19]
```diff
@@ -3,6 +3,309 @@
...
+ruins with the Xenoresurrection Lab
+- Added the new special Temporal Anomaly
+- Reworked weapon part refinements, so that instead of multiple parts,
+techs increase the strength of parts (although getting the increase to
+take effect requires being in supply range)
+- Added Energy Frigate hull, adjusted some other hulls' balance accordingly
+- Updated premade / default ship designs, made naming consistent
+- New bombardment weapons (feature in-progress)
+- Added EMP and electric overcharge ship parts
+- Added Solar Concentrators which increase laser weapon strength
```
Comment (MatGB) [inline ChangeLog.md]: for Organic Line hulls

File: ChangeLog.md (hunk 11 above)
Comment (MatGB) [reply ChangeLog.md]: "At start of game, the Honeycomb will now turn all planets without specials or inhabitants within 5 jumps into Asteroids instead of destroying them—without renaming them"

It's user facing because the weirdly named asteroids and similar is a deliberate hint, and not destroying the planets ties in with the "all-stars-get-a-planet" change.

File: ChangeLog.md (hunk 13 above)
Comment (MatGB) [reply ChangeLog.md]: agreed, it was IIRC done in the same PR but as a different commit.

File: ChangeLog.md [hunk 20]
```diff
@@ -3,6 +3,309 @@
...
+- Tweaked monster detection and stealth stats
+- Artificial Moons can no longer be built when a Resonant Moon is present
+- Reworked various modifiers to planet population
+- Tweaked Honeycomb and World Tree special locations
+- Re-Added the Tidally Locked Rotation special with a population
+penalty and changed the penalty of the Eccentric Orbit special to a
+Supply minus
+- Space Monster weapon balance tweaks
+- Planetary Starlane Drive modified so it can't be built on gas giants
+- Xenoresurrection Lab building modified so that only one can be built
```
Comment (MatGB) [inline ChangeLog.md]: This either doesn't need to be here or should be alongside the line mentioning their introduction

File: ChangeLog.md [hunk 21]
```diff
@@ -3,6 +3,309 @@
...
+on a planet
+- Death Spores and Bio-Terminators now only affect organic species
+- Monsters now only bombard one planet at a time
+- Adjusted Experimentor monster weapons
+- Reworked how derelict visibility effect works - grants visibility
+directly instead of temporarily increasing ship detection range
```
Comment (MatGB) [inline ChangeLog.md:239]: I haven't seen a mention of telepathic detection for Trith yet, they use the same mechanics.

**Comment by MatGB** [comment]
The Drydock change needs to be up as a Key Change I think, they're a signficant part of the game. I would also consider putting the damage control changes, Basic is changed, Robotic hulls are the most used hulls and the robotic line in general is one of the most used.

"Ship Repair has been substantially reworked, Drydocks require a ship to be stationary and repair a proportion of ship structure, no damage control effects work on the turn a combat has taken place except for the Logistic Facilitator flagship, which is substantially reduced."

**Comment by MatGB** [comment]
The change to directory structure on storage is important to note (possibly as a separate line item below key changes), as it still confuses me on occasions and we know there are a fair few linux users that only get their copy of the game from the official repos. When they update to this their save files and config files will have moved unexpectedly.

If it's not mentioned, the change to the directory structure for scripting needs to be mentioned, there are enough amateur scripters and similar that might have their own hacks, etc.

File: ChangeLog.md (hunk 17 above)
Comment (geoffthemedio) [reply ChangeLog.md:21]: That's what "GUI responsiveness / FPS improvements" means.

File: ChangeLog.md (hunk 18 above)
Comment (geoffthemedio) [reply ChangeLog.md:83]: Disagree. It's not a major new feature or redesign, and the current implementation in the GUI is rather awkward to emphasize it so much.

File: ChangeLog.md (hunk 20 above)
Comment (geoffthemedio) [reply ChangeLog.md]: reasonable

File: ChangeLog.md (hunk 21 above)
Comment (geoffthemedio) [reply ChangeLog.md:239]: reasonable

**Comment by geoffthemedio** [comment]
Drydock/repair -> ok
Directory structure -> seems weird to mention as key, but if you insist, ok...

File: ChangeLog.md (hunk 15 above)
Comment (geoffthemedio) [reply ChangeLog.md]: Details are explained below. The line about it can be moved to the top of the balance section.

File: ChangeLog.md (hunk 11 above)
Comment (Vezzra) [reply ChangeLog.md]: @MatGB has a point here I think, so I'll put in the text he suggested for now.

File: ChangeLog.md (hunk 8 above)
Comment (Vezzra) [reply ChangeLog.md]: No disabling of any content at this point, unless required for a fix. There is no harm in keeping these parts, and it's not worth the risk of breaking something.

However, I don't think it's necessary to specifically mention them, even if used by some space monsters.

File: ChangeLog.md (hunk 15 above)
Comment (Vezzra) [reply ChangeLog.md]: Agree with @geoffthemedio here. A concise summary is sufficient for the "Key Changes" section, the details would add to much clutter here.

File: ChangeLog.md (hunk 16 above)
Comment (Vezzra) [reply ChangeLog.md:24]: Yep, that has already been pointed out by @Morlic-fo too, and was also on my mind.

File: ChangeLog.md (hunk 17 above)
Comment (Vezzra) [reply ChangeLog.md:21]: Yep, agree with @geoffthemedio, "GUI responsiveness / FPS improvements" covers that sufficiently.

File: ChangeLog.md (hunk 15 above)
Comment (MatGB) [reply ChangeLog.md]: I'd missed it reading through, I think perhaps a (details below) bracket or movign it to the top of its section as Geoff suggests would also be useful)

File: ChangeLog.md (hunk 17 above)
Comment (MatGB) [reply ChangeLog.md:21]: See, I'm not sure it's clear enough for what it is. "GUI responsiveness / FPS improvements / Lag reduction" would I think be better, it's been such a problem for some players (including me) for so long that it needs to be clear that we've substantially reduced it. Given I didn't understand that that point was saying that, it won't just be me that misses it.

**Comment by MatGB** [comment]
Heh, I never understood why it was changed—it's some Linux standardisation thing, but given it really does affect you if you're on Linux and used to looking at config files for the game, etc, it needs to be clearly stated somewhere. I don't think it's a key change but I do think we're going to get some confused players.

File: ChangeLog.md (hunk 17 above)
Comment (geoffthemedio) [reply ChangeLog.md:21]: I don't think adding "Lag reduction" clarifies much, as the term has several meanings, and the default you use it as (GUI responsiveness) is not my default (network latency). I specifically put "GUI responsiveness" in to address... well... GUI responsiveness.

**Comment by geoffthemedio** [comment]
Add a note in technical / other about directory changes? (if not already suggested...)

File: ChangeLog.md (hunk 17 above)
Comment (Vezzra) [reply ChangeLog.md:21]: Well, maybe the line doesn't convey sufficiently just how substantial the improvements are, so maybe rephrase it like this: `Substantial improvements to GUI responsiveness and FPS`? That ok for both of you?

**Comment by Vezzra** [comment]
I think proper entries in technical for both the restructuring of the content scripts and this thing with the config files on Linux should be sufficient. If anyone of our English native speakers could come up with some properly phrased texts, I'd very much appreciate it (as a non-native speaker it's a bit more difficult for me... 😉).

**Comment by Vezzra** [comment]
> Damage control techs that only affect a subset of ship types (robotic?) is probably a bit narrow for a key change

Only a sidenote: That revision _does_ affect all ships, not only the robotic, those were just part of it because of their inherent self-repair capabilities.

File: ChangeLog.md (hunk 17 above)
Comment (geoffthemedio) [reply ChangeLog.md:21]: as you want

File: ChangeLog.md [hunk 22]
```diff
@@ -3,6 +3,309 @@
...
+- Made producing troop pods require a production location with troops,
+and made producing troop pods consume troops from the production
+location
+ - Species with no offensive Ground Troops will no longer be able to
+build ships with Ground Troop pods
+- Balanced research and production bonuses
+- Reprioritised mine effects to always occur before repair effects
+- Balance pass on Robotic line and Damage Control techs
+- Balanced BioAdaptive hull to work same as Nanorobotic
+- Tweaked numbers for Logistics Facilitator
```
Comment (Vezzra) [inline ChangeLog.md:247]: @MatGB: the above 4 entries ("Reprioritised mine effects to always occur before repair effects" to "Tweaked numbers for Logistics Facilitator") all seem to have to do/potentially have to do with the ship repair revision. Which of these should be grouped with the bigger entry dealing with that revision? Should they be rephrased/rearranged when doing this, if yes, any suggestions?

Maybe wait until I commit all the changes/adjustments suggested so far, then you can review the my draft of the ship repair revision entry and provide your input/feedback/adjustments.

**Comment by dbenage-cx** [comment]
`- (*nix) Moved user config and data files to conform with XDG Base Directory Specification`
`- Reorganized content definitions into multiple files`

`*nix` is a bit unspecific, but it gets the point across to Win/OSX not to worry about it, while letting the various distro users note it _likely_ applies to them.

**Comment by Vezzra** [comment]
Thanks, looks good enough for me. 👍

**Comment by Vezzra** [comment]
@MatGB:

> The Drydock change needs to be up as a Key Change I think, they're a signficant part of the game. I would also consider putting the damage control changes, Basic is changed, Robotic hulls are the most used hulls and the robotic line in general is one of the most used.
> 
> "Ship Repair has been substantially reworked, Drydocks require a ship to be stationary and repair a proportion of ship structure, no damage control effects work on the turn a combat has taken place except for the Logistic Facilitator flagship, which is substantially reduced."

I've added a line to the key changes, and tried to craft your paragraph into an entry+subentries in the "Balance" section for details. Please review.

File: ChangeLog.md [hunk 23]
```diff
@@ -3,6 +3,316 @@
...
+
+
+### Key Changes
+
+- Reworked supply mechanics (details below)
+- Reworked weapon part refinements
+- Reworked ship repair: Drydocks, Damage Control techs,
+Robotic hulls self repair (details below)
+- Substantial GUI improvements
+ - Substantial improvements to GUI responsiveness and FPS
```
Comment (Vezzra) [inline ChangeLog.md:16]: @MatGB: do you think that's clear enough now...?

File: ChangeLog.md [hunk 24]
```diff
@@ -3,6 +3,316 @@
...
+of current save output format setting
+- Made communication between clients / servers on different operating
+systems more flexible / robust
+- Reworked XML save format to have an uncompressed somewhat
+readable-as-text header with basic info about the saved game
+- Added --version parameter to freeorion{,d} executables which outputs
+the version string to the console
+- (*nix) Moved user config and data files to conform with
+XDG Base Directory Specification
+- Reorganized content definitions into multiple files
```
Comment (Vezzra) [inline ChangeLog.md:314]: Ok, very concise, but not sure if that's really, hm, sufficiently "prominent"... if anyone has better ideas, speak up 😉

**Comment by Vezzra** [comment]
Alright, big update has been committed, I've tried to incorporate all suggestions where we've reached at least some kind of consensus. Thanks to everyone for their input! 😃 

Haven't been sure on the best phrasing/summary of some things, may have missed/misunderstood others, and on some items the consensus wasn't, well, uncontested. 😉

So again, everyone please review the changes and give your feedback/corrections.

File: ChangeLog.md [hunk 25]
```diff
@@ -250,6 +250,8 @@ build ships with Ground Troop pods
 artificial blackhole
 - Prevent Experimentor spawn location from sundering the galaxy
 - Balance Hyperspatial Dam
+- Made planets in systems which contains a Gateway to the Void building
```
Comment (geoffthemedio) [inline ChangeLog.md]: systems that contain or a system that contains

Comment (Vezzra) [reply ChangeLog.md]: Oops, thanks!

File: ChangeLog.md (hunk 23 above)
Comment (MatGB) [reply ChangeLog.md:16]: Yup, that's good.

Comment (geoffthemedio) [reply ChangeLog.md:16]: I don't think we need "substantial" "GUI" and "improvements" in the main and sub-point

File: ChangeLog.md [hunk 26]
```diff
@@ -3,6 +3,317 @@
...
+- Reworked supply mechanics (details below)
+- Reworked weapon part refinements
+- Reworked ship repair: Drydocks, Damage Control techs,
+Robotic hulls self repair (details below)
+- Substantial GUI improvements
+ - Substantial improvements to GUI responsiveness and FPS
+ - Made GUI more configurable
+- Many small additions/changes to GUI rendering, layout, functionality,
+customization options, and displayed information that collectively
+substantially improve usability, responsiveness, and system resource usage.
```
Comment (geoffthemedio) [inline ChangeLog.md:20]: responsiveness again...?

Just use this point, not the parent or sibling

File: ChangeLog.md (hunk 23 above)
Comment (Vezzra) [reply ChangeLog.md:16]: Ok, will remove it from the main point.

File: ChangeLog.md (hunk 26 above)
Comment (Vezzra) [reply ChangeLog.md:20]: I don't quite understand what you mean? That line has been your suggestion...

Comment (geoffthemedio) [reply ChangeLog.md:20]: I don't see the need for a point for "Substantial GUI improvements" with sub-points about "Substantial improvements to GUI" and "Made more configurable", and then a separate point for "Many small additions/changes to GUI ..." where the second point pretty much covers what the first one and its sub-points say...

Seems oddly repetitive...

Comment (Vezzra) [reply ChangeLog.md:20]: @geoffthemedio, to save time (I'm deferring the RC1 builds here), just commit the corrections/adjustments as you see fit.

Comment (Vezzra) [reply ChangeLog.md:20]: Ah, now I get it. Apparently I misunderstood what you meant when you originally suggested that line - I thought it should replace just the one sub-point, but you meant to replace the entire main+sub points. Ok, will make the requested changes.

**Comment by geoffthemedio** [comment]
I'm not somewhere where I can do that, I think...

**Comment by Vezzra** [comment]
No problem, I finally got what you meant and will remove the other point (the one with a main+subpoints).
